python/python/bystro/search/utils/messages.py

```diff
@@ -30,6 +61,7 @@ class SaveJobData(BaseMessage, frozen=True):
     indexName: str
     outputBasePath: str
     fieldNames: list[str]
+    pipeline: list[BinomialMafFilter | HWEFilter] | None = None
```

Comment:
if the semantics of `None` here are equivalent to an empty list of filters, we could reduce this type to `list[BinomialMafFilter | HWEFilter]` and avoid explicit checks for `None`